explorer/utils.py

```diff
@@ -2,12 +2,7 @@
 import re
 
 from django.contrib.auth.forms import AuthenticationForm
-
-try:
-    from django.contrib.auth.views import login
-except ImportError:
-    pass
-
+from django.contrib.auth.views import LoginView
```

Comment:
That breaks backwards compatibility right ?
Can we make the code working on previous django version too?

Reply:
It was introduced django 1.11 earlier versions of django aren't supported by django foundation anymore.

Reply:
https://docs.djangoproject.com/en/1.11/topics/auth/default/#django.contrib.auth.views.login

Reply:
Cool, can we make tests passing by removing deprecated django versions ?